Drop-In/src/main/java/com/braintreepayments/api/CardDetailsFragment.java

```diff
@@ -195,4 +200,15 @@ public void onCardFormFieldFocused(View view) {
             sendDropInEvent(DropInEvent.createEditCardNumberEvent(cardNumber));
         }
     }
+
+    private void hideSoftKeyboard() {
+        // Ref: https://stackoverflow.com/a/3553811
+        FragmentActivity activity = getActivity();
+        if (activity != null) {
+            InputMethodManager imm =
+                    (InputMethodManager)activity.getSystemService(Context.INPUT_METHOD_SERVICE);
+            View focusedView = activity.getCurrentFocus();
+            imm.hideSoftInputFromWindow(focusedView.getWindowToken(), 0);
+        }
+    }
```

Comment:
I couldn't find a good way to test keyboard dismissal, but this will ensure that the loader is visible after "Done" is pressed.